src/http/http_thread.zig

```diff
@@ -0,0 +1,481 @@
+var custom_ssl_context_map = std.AutoArrayHashMap(*SSLConfig, *NewHTTPContext(true)).init(bun.default_allocator);
+
+pub const HTTPThread = struct {
```

Comment:
The file name should be `HTTPThread.zig`, and this should be a top-level struct
```suggestion
const HTTPThread = @This();
```